test_conformance/basic/test_imagearraycopy.cpp

```diff
@@ -29,7 +29,7 @@ int test_imagearraycopy_single_format(cl_device_id device, cl_context context, c
   clMemWrapper      image, buffer;
   int        img_width = 512;
   int        img_height = 512;
-  size_t    elem_size;
+  size_t    elem_size; 
```

Comment:
This change shouldn't be necessary and causes the formatting check to fail now.   Other than this, changes in this file look good to me.

Reply:
Sorry, that should be a typo. Removed the white space.